OVRControllerGamepadMac could not be found

## Describe the bug
I created a new blank project with latest xrtk packages installed.
Trying to build project console shows error:
`Library/PackageCache/com.xrtk.oculus@3ae186b0f5bc2ddcd455bd18dacc3620338f7268/OculusApi.cs(78,20): error CS0246: The type or namespace name 'OVRControllerGamepadMac' could not be found (are you missing a using directive or an assembly reference?)`

Removed and added all packages and installed Oculus Desktop/Android 1.38.1, still same error


## Information(please complete the following information):
 - Version: 2019.1.10f1 **MacOs**

--- a/XRTK.Oculus/Packages/com.xrtk.oculus/OculusApi.cs
+++ b/XRTK.Oculus/Packages/com.xrtk.oculus/OculusApi.cs
@@ -943,6 +943,14 @@ namespace XRTK.Oculus
             }
         }
 
+        private class OVRControllerGamepadMac : OVRControllerBase
+        {
+            public OVRControllerGamepadMac()
+            {
+                controllerType = Controller.Gamepad;
+            }
+        }
+
         private class OVRControllerGamepadAndroid : OVRControllerBase
         {
             public OVRControllerGamepadAndroid()

